Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -23,5 +23,6 @@
 numWFIB.extend([21234.0]) #2026D44
 numWFIB.extend([21634.0]) #2026D45
 numWFIB.extend([22034.0]) #2026D46
+numWFIB.extend([22434.0]) #2026D47
```

Comment:
@bsunanda @kpedro88 this inflates further the list of geometry scenarios that we would regularly run in tests. Do we need at the same time D45, D46 and D47? Do you have a plan for the scenarios evolution and final target?

Reply:
It is not included in the PR tests, only the IB tests. To me, this is reasonable. We will try to keep the PR tests to a minimum, but the IB should test every workflow (in my opinion).